redfish-core/lib/log_services.hpp

```diff
@@ -1413,6 +1462,34 @@ inline void requestRoutesJournalEventLogEntry(App& app)
             });
 }
 
+static bool isHiddenEventLogEntry(std::string& path)
+{
+    constexpr auto loggingService = "xyz.openbmc_project.Logging";
+    constexpr auto loggingPelInterface = "org.open_power.Logging.PEL.Entry";
+    constexpr auto loggingPelProperty = "Hidden";
+
+    constexpr auto interface = "org.freedesktop.DBus.Properties";
+
+    auto bus = sdbusplus::bus::new_default();
+    bool propValue = false;
+    try
```

Comment:
Why the try / catch? 
Let's follow https://github.com/openbmc/bmcweb/blob/83fd8e19867f43f3a175d1ede65ad8636da553b3/redfish-core/lib/systems.hpp#L673

Reply:
I needed the try block since the org.open_power.Logging.PEL.Entry interface and 'Hidden' property weren't there in older LogEvent entries - that is the event entries created before the Hidden property feature was added. It was coming out with (internal?) error, so just added the try...catch block to avoid such error and also to let those event log entries printed. 
I can remove this if it is assumed that all event log entries will have Hidden proprty...

Reply:
Lets do something like


```
{
    BMCWEB_LOG_DEBUG << "Get host information.";
    crow::connections::systemBus->async_method_call(
        [aResp](const boost::system::error_code ec,
                const std::variant<std::string>& hostState) {
            if (ec)
            {
                BMCWEB_LOG_DEBUG << "DBUS response error " << ec;
                messages::internalError(aResp->res);
                return;
            }
            
            ...
            

        },
        "xyz.openbmc_project.Logging", "/xyz/openbmc_project/state/host0",
        "org.freedesktop.DBus.Properties", "Get",
        "org.open_power.Logging.PEL.Entry", "Hidden");
}
```

Look at like https://github.com/openbmc/bmcweb/blob/83fd8e19867f43f3a175d1ede65ad8636da553b3/redfish-core/lib/systems.hpp#L673